stestr/commands/run.py

```diff
@@ -36,6 +36,8 @@
 
 def _to_int(possible, default=0, out=sys.stderr):
     try:
+        if possible is None:
+            raise ValueError(possible)
         i = int(possible)
     except ValueError:
```

Comment:
Couldn't we just add ValueError to this exception with something like:

```suggestion
    except (ValueError, ValueError):
```

instead of converting the exception type?

Reply:
Ah, yeah, that must be simpler :) I'll update it soon.